package.json

```diff
@@ -35,6 +32,7 @@
   "dependencies": {
     "@nestjs/common": "^5.0.0",
     "reflect-metadata": "^0.1.12",
-    "rxjs": "^6.1.0"
+    "rxjs": "^6.1.0",
+    "rxjs-compat": "^6.2.1"
```

Comment:
both `rxjs` and `rxjs-compat` have to stay as `devDependencies` (otherwise, it will create side-effect, for instance, incorrectly working `instanceof` operator).